### PR TITLE
Blog post: opensearch supported by tracetest

### PR DESCRIPTION
**Pull request by mathnogueira**
Signed-off-by: Matheus Nogueira <matheus.nogueira2008@gmail.com>

### Description
The blog post about OpenSearch now being supported by Tracetest and how one can configure a Tracetest instance to use OpenSearch as its tracing backend.
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by krisfreedain** [comment]
This is fantastic @mathnogueira! Thank you for this, as well as https://github.com/opensearch-project/project-website/pull/1024/ so we can get Tracetest/Kubeshop added to the partner page. I've opened a quick request with our editorial department for their review and we'll get back to you ASAP. Thank you!

File: _posts/2022-09-27-opensearch-supported-by-tracetest.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+### Installing Tracetest with OpenSearch
+
+The Tracetest team is working diligently to make installation as seamless as possible. Currently, you can use CLI to install the server to run on either Docker Compose or Kubernetes. Visit the [installation guide](https://tracetest.io/download) to install both the CLI and server.
+
+Before proceeding, make sure you have an OpenSearch instance running and have a Data Prepper instance pointing to it. Data Prepper is an OpenSearch component capable of converting OpenTelemetry format into JSON that can be indexed by OpenSearch. Tracetest has an [example application](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-opensearch) where you can see how the Data Prepper is configured.
+
+For simplicity sake in this article, you will learn how to setup Tracetest using Docker Compose. Execute the following steps to configure your Tracetest instance to connect to your OpenSearch instance:
```
Comment (jhmcintyre) [inline _posts/2022-09-27-opensearch-supported-by-tracetest.md:23]: suggest "For simplicity's sake"

File: _posts/2022-09-27-opensearch-supported-by-tracetest.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+![Empty assertions](/assets/media/tutorials/tracetest/adding_assertions.png){: .img-fluid}
+
+Click on **Save Test Spec** and then on **Publish**.
+
+Now it's possible to run the test. Tracetest will execute all assertions you just created using the new trace. Click on **Run Test** and wait for the test to finish executing.
+
+![Adding assertions](/assets/media/tutorials/tracetest/test_rerun.png){: .img-fluid}
+
+And Tracetest is not only an UI tool. You also can run that test from your CI environment. Click on the gear icon at the top of the page and click on **Test Definition**.
```
Comment (jhmcintyre) [inline _posts/2022-09-27-opensearch-supported-by-tracetest.md:99]: suggest revise first sentence to "Tracetest is not only a UI tool."

File: _posts/2022-09-27-opensearch-supported-by-tracetest.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+layout: post
+title:  "Using OpenSearch as the traces storage for Tracetest"
+authors:
+  - mathnogueira
+date:   2022-09-27 12:15:00 -0300
+categories:
+  - community
+---
+
+It is exciting to announce that [Tracetest](https://tracetest.io) now supports OpenSearch! If you already use OpenSearch Trace Analytics, you can start writing tests based on your telemetry using Tracetest without having to change anything in your application. In this article, we will explain how you can start building trace-based tests right now.
```
Comment (jhmcintyre) [inline _posts/2022-09-27-opensearch-supported-by-tracetest.md:11]: suggest revise to "I am excited to announce that"

File: _posts/2022-09-27-opensearch-supported-by-tracetest.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+### Installing Tracetest with OpenSearch
+
+The Tracetest team is working diligently to make installation as seamless as possible. Currently, you can use CLI to install the server to run on either Docker Compose or Kubernetes. Visit the [installation guide](https://tracetest.io/download) to install both the CLI and server.
+
+Before proceeding, make sure you have an OpenSearch instance running and have a Data Prepper instance pointing to it. Data Prepper is an OpenSearch component capable of converting OpenTelemetry format into JSON that can be indexed by OpenSearch. Tracetest has an [example application](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-opensearch) where you can see how the Data Prepper is configured.
```
Comment (jhmcintyre) [inline _posts/2022-09-27-opensearch-supported-by-tracetest.md:21]: in last line, you can just say "see how Data Prepper is configured."

**Review by jhmcintyre — COMMENTED** [review]
Please see comments for suggested revisions for clarity/usage.

**Comment by mathnogueira** [comment]
Thanks for the suggestions @jhmcintyre! I just applied them.

**Comment by krisfreedain** [comment]
thanks @jhmcintyre for the review and thanks @mathnogueira for the post and quick turnaround on edits.

**Review by krisfreedain — APPROVED** [review]

**Pull request merged**